File: app/src/main/java/com/decode/tumblr/model/Note.java
package com.decode.tumblr.model;

import com.google.gson.annotations.SerializedName;

public class Note {

    @SerializedName("type")
    public String type;
    @SerializedName("timestamp")
    public Integer timestamp;
    @SerializedName("blog_name")
    public String blogName;
    @SerializedName("blog_uuid")
    public String blogUuid;
    @SerializedName("blog_url")
    public String blogUrl;
    @SerializedName("followed")
    public Boolean followed;
    @SerializedName("avatar_shape")
    public String avatarShape;
    @SerializedName("post_id")
    public String postId;
    @SerializedName("reblog_parent_blog_name")
    public String reblogParentBlogName;

}

File: app/src/main/java/com/decode/tumblr/data/model/Note.kt
package com.decode.tumblr.data.model

import com.google.gson.annotations.SerializedName

data class Note(

        @SerializedName("type")
        var type: String? = null,

        @SerializedName("timestamp")
        var timestamp: Int? = null,

        @SerializedName("blog_name")
        var blogName: String? = null,

        @SerializedName("blog_uuid")
        var blogUuid: String? = null,

        @SerializedName("blog_url")
        var blogUrl: String? = null,

        @SerializedName("followed")
        var followed: Boolean? = null,

        @SerializedName("avatar_shape")
        var avatarShape: String? = null,

        @SerializedName("post_id")
        var postId: String? = null,

        @SerializedName("reblog_parent_blog_name")
        var reblogParentBlogName: String? = null

)
